src/app/pages/about/about.component.html

```diff
@@ -0,0 +1,30 @@
+<section class="container">
+    <h1>About The <span>Team</span></h1>
+    <p>
+        Chef’s Compass is a recipe finding app for you, the user. It helps you find a recipe based on your
+        hunger-tackling desires and any ingredients stocked in your kitchen. Not sure how to use them effectively? Let
+        Chef’s Compass show you the way.
+      </p>
+
+<div class="team">
+    <section class="team_member" *ngFor="let team of teams" >
```

Comment:
your ng for will generate a section for each member, put at the beginning of the member card.
Create an ul an li as well for better referencing